app/scripts/services/dimSettingsService.factory.js

```diff
@@ -83,6 +84,15 @@
       // self destruct timer for quality, hope we can remove this some day...
       savedSettings.disableQuality = new Date('9/19/2016') <= new Date();
 
+      // for now just override itemTags. eventually let users create own?
```

Comment:
I needed to override the itemTags[] above because the above array was just the default. 

With custom notes... do we need custom tags? I could be persuaded either way. What other tags would people be interested in adding, that they couldn't just have in the notes section?